x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -35,6 +36,11 @@ func (k Keeper) HandleAcknowledgement(ctx sdk.Context, modulePacket channeltypes
 	err := json.Unmarshal(acknowledgement, &ack)
 	if err != nil {
 		ackErr := channeltypes.Acknowledgement_Error{}
+		// Clean up any pending claims
+		_, found := k.GetPendingClaims(ctx, packetSequenceKey)
+		if found {
+			k.RemovePendingClaims(ctx, packetSequenceKey)
```

Comment:
is the idea here to remove pending status on ack errors so that we can re-attempt to claim them in the next day epoch?

Reply:
makes sense!